src/bci_build/package/sle15-kernel-module-devel/README.md.j2

```diff
@@ -0,0 +1,56 @@
+# SLE {{ image.os_version.pretty_os_version_no_dash }} Kernel Module Development Container
+{% include 'badges.j2' %}
+
+## Description
+
+The SLE {{ image.os_version.pretty_os_version_no_dash }} Kernel Module
+Development Container Image ships all necessary tools to build kernel modules
+for SLE and SLE Micro. It is intended to be used for building out of tree kernel
+modules in restricted environments for the SLE kernel.
```

Comment:
```suggestion
for SLE and SLE Micro. It is intended to be used for building out of tree kernel
modules in restricted environments for the SLE kernel.
```

_building out of tree kernel modules_ Can you _build out of_? From the grammar point of view, it sounds strange.

Reply:
That's the technical term. Most kernel modules are in the kernel source tree (i.e. in Linus Torvalds git repo), but some are developed separately and hence are called "out of tree"

Reply:
I think grammar wise `out-of-tree kernel modules` would be more correct and easier to read.